tests/python/test_with_shap.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import xgboost as xgb
+import testing as tm
+import unittest
+import pytest
+
+try:
```

Comment:
Can we move this into testing.py?

Reply:
I prefer it to stay here, I don't think it will be reused.